README.md

```diff
@@ -622,7 +622,7 @@ Other Style Guides
     });
     ```
 
-  - [8.2](#8.2) <a name='8.2'></a> If the function body consists of a single expression, feel free to omit the braces and use the implicit return. Otherwise use a `return` statement.
+  - [8.2](#8.2) <a name='8.2'></a> If the function body consists of a single expression, feel free to omit the braces and use the implicit return. Otherwise add the parentheses, braces, and use a `return` statement.
```

Comment:
8.2 is about braces. 8.4 is about parentheses.

How about saying “This rule does not apply to functions with a multiline body wrapped in braces or parentheses. If your function spans over multiple lines, always wrap its arguments in parentheses.” in 8.2 then?

Then we can drop “feel free” and “add the parentheses” from this rule – and we’re done!